static/app/components/events/groupingInfo/groupingInfo.tsx

```diff
@@ -79,7 +77,7 @@ export default function GroupingInfo({
 
   return (
     <Fragment>
-      {hasStreamlinedUI && (
+      {(isSuccess || hasPerformanceGrouping) && (
         <GroupInfoSummary event={event} group={group} projectSlug={projectSlug} />
```

Comment:
would be nice to having a placeholder here so the rest of the grouping information doesn't get shifted down when the info summary is shown